tests/python/python-peewee-mysql-empty-password-python-test.yml

```
@@ -5,4 +5,4 @@ valid:
 invalid:
   - |
     mysql_db1 = MySQLDatabase('my_app', user='app', password='', host='10.1.0.8', port=3306) 
- 
\ No newline at end of file
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)